demos/serverless/src/handlers.js

```diff
@@ -312,4 +313,27 @@ function addSignalMetricsToCloudWatch(logMsg, meetingId, attendeeId) {
       putMetric(metricName, metricValue, meetingId, attendeeId);
     }
   }
+}
+
+function addEventIngestionMetricsToCloudWatch(log, meetingId, attendeeId) {
+  const putMetric =
+    metricScope(metrics => (metricName, metricValue, meetingId, attendeeId) => {
+      metrics.setProperty('MeetingId', meetingId);
+      metrics.setProperty('AttendeeId', attendeeId);
+      metrics.putMetric(metricName, metricValue);
+    });
+  const { logLevel, message } = log;
+  let errorMetricValue = 0;
+  let retryMetricValue = 0;
+  let ingestionTriggerSuccessMetricValue = 0;
+  if (logLevel === 'ERROR') {
+    errorMetricValue = 1;
```

Comment:
Can we just call putMetric right here? Why do we need these variable?

Reply:
I ran into errors when no metric was posted so tried to this way like 1 when observed 0 when not.